src/SDKs/CustomerInsights/Management.CustomerInsights/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CustomerInsights Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure CustomerInsights management functions for managing the Microsoft Azure CustomerInsights service.")]
 
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.9.1.0")]
```

Comment:
@tjlvtao why there is such huge difference between assembly and file version? Can this be made consistent.